pkgs/top-level/release-small.nix

```diff
@@ -2,7 +2,7 @@
    the load on Hydra when testing the `stdenv-updates' branch. */
 
 { nixpkgs ? { outPath = (import ../../lib).cleanSource ../..; revCount = 1234; shortRev = "abcdef"; }
-, supportedSystems ? [ "x86_64-linux" "i686-linux" "x86_64-darwin" ]
+, supportedSystems ? [ "x86_64-linux" "x86_64-darwin" ]
```

Comment:
That isn't what I was expecting, but we could. I was expecting nixos-unstable to block on i686 but no -small's, and no wide package tree -- just essentials.